documentation/api/admin/v1/archive.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: "PuppetDB 3.0 » Admin API » v1 » Importing and Exporting PuppetDB Archives"
+layout: default
+canonical: "/puppetdb/latest/api/admin/v1/archive.html"
+---
+
+[curl]: ../curl.html#using-curl-from-localhost-non-sslhttp
+
+The `/archive` endpoint can be used for importing and exporting PuppetDB archives.
+
+## `POST /pdb/admin/v1/archive`
+
+This endpoint can be used for streaming a PuppetDB archive into PuppetDB.
+
+### Request Format
+
+The request should be a multipart POST and have `Content-Type: multipart/mixed`.
+
+### URL Parameters
+
+* `archive`: Required. The archive file to import to the PuppetDB.
+* `command_versions`: Required. A JSON object mapping PuppetDB command-names to their version.
```

Comment:
We should say something about what to use here, for exports from various versions of PuppetDB.